### PR TITLE
Feature/add basic reduction to manual

### PR DESCRIPTION
**Pull request by abostroem**

### CODE REVIEW

File: manual.md [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 * [Running the pipeline](#Running-the-pipeline)
+* [Cookbook](#Cookbook)
 * [Creating a Landolt Catalog](#Creating-a-Landolt-Catalog)
 * [Difference Imaging](#Difference-Imaging)
 * [Definitions](#Definitions)
 
 # Ingest data
-* Download observations from SNEx and run ingesttar.py
-* Download observations from archive.lco.global using LCOGTingest.py. This will only work for public data (e.g., standard stars images and observations from over 1 year ago) unless you have an account for the observing portal. For example (all arguments optional; see ```LCOGTingest.py -h``` for options):
+* Download observations from SNEx and run ingesttar.py. This data will have already been reduced with BANZAI.
```
Comment (cmccully) [inline manual.md:11]: Will this data also have been reduced by snex at all? Is this for floyds data?

Comment (abostroem) [reply manual.md:11]: This is only photometry and my understanding was that the images you download as a tar file from SNEx is reduced

File: manual.md [hunk 2]
```diff
@@ -17,6 +18,54 @@ LCOGTingest.py -n NAME -s YYYY-MM-DD -e YYYY-MM-DD -t EXPOSE -r reduced --public
...
+
+1. Verify your PSF catalog. It is recommended that the `psf` stage be run with the catalog option pointing to one of the catalogs you downloaded (usually apass or sloan) so that the same stars are used for build the PSF for every image. You should run the `psf` stage using the `--show` option on a few images in each filter to verify that the star selected from the catalog are good (e.g. not double stars, near a hot pixel, or saturated). In addition to showing you the model PSF, if a DS9 window is open prior to running, the image will be shown with the stars using to build the PSF circled in cyan. Note that these ID number represent the line number (after the header) in the catalog file and not the catalog ID numbers. Remove any stars that are being selected and are not good from the catalog in `$LCOSNPIPE/trunk/src/lsc/standards/cat/...`. Additionally, you copy the file to the installation directory. The easiest way to do this is to re-install the pipelines `python setup.py install`. Repeat this stage until you are satisfied with the stars selected for the catalog.
```
Comment (cmccully) [inline manual.md:25]: Is the catalog downloaded as part of the installation?  Or does that need to be part of this step?

Comment (cmccully) [reply manual.md:25]: I would also make the prescriptive to open DS9 first.

Comment (abostroem) [reply manual.md:25]: its part of the previous step - line 23 says it assumes you've done this already

**Pull request merged**